pkg/controller/nodepool/nodepool_controller.go

```diff
@@ -141,51 +130,30 @@ func (r *ReconcileNodePool) Reconcile(ctx context.Context, req reconcile.Request
 	// Note !!!!!!!!!!
 	// We strongly recommend use Format() to  encapsulation because Format() can print logs by module
 	// @kadisi
-	klog.Infof(Format("Reconcile NodePool %s/%s", req.Namespace, req.Name))
+	klog.Infof(Format("Reconcile NodePool %s", req.Name))
```

Comment:
No need to output namespace?

Reply:
yeah, because NodePool is cluster scope resource, so namespace is not set.